iptables.go

```diff
@@ -36,6 +36,13 @@ func newIPTables(config *bouncerConfig) (interface{}, error) {
 		CheckIptableCmds: [][]string{},
 	}
 
+	var target string
+	if config.DenyAction == "reject" {
```

Comment:
maybe perform case insensitive match with `strings.EqualFold` and then normalized to "REJECT" / "DROP" here ?   It would avoid "REJECT" being interpreted as "DROP" here.

Reply:
yes, that seems like a good idea, updated now